tools/universe_builder.py

```diff
@@ -92,6 +92,9 @@ def _create_tree(self, scratchdir):
         pkgdir = os.path.join(treedir, 'repo', 'packages', self._pkg_name[0].upper(), self._pkg_name, '0')
         os.makedirs(pkgdir)
         for pkgfile in os.listdir(self._input_dir_path):
+            # Don't copy in .dotfiles; editor temps etc
+            if pkgfile.startswith('.'):
```

Comment:
Yes this does not affect .stub_dir.  The .stub_dir is created in the target location higher in the function.  This is simply copying framework/*/universe into the target.  copyfile would fail anyway on a directory.